server/Customer.java

```diff
@@ -62,4 +64,13 @@ public String getUsername() {
 	public String getPassword() {
 		return password;
 	}
+
+	public void setAccountLocked(boolean value){
+		this.accountLocked = value;
```

Comment:
Just a nitpick, but you said you wanted me to give you comments :)

Similar to how you used `System.out` and `out`, here you use `this.accountLocked` and `accountLocked`, decide for one and stay consistent